kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -800,6 +801,19 @@ def parse(s: str) -> list[T]:
     kevm_kompile_args.add_argument(
         '-o', '--output-definition', type=Path, dest='output_dir', help='Path to write kompiled definition to.'
     )
+    kevm_kompile_args.add_argument(
+        '--with-llvm-library',
+        dest='llvm_library',
+        default=False,
+        action='store_true',
+        help='Do not generate a dynamic llvm library during kompile.',
+    )
+    kevm_kompile_args.add_argument(
+        '--no-llvm-library',
+        dest='llvm_library',
+        action='store_false',
+        help='Make kompile generate a dynamic llvm library.',
```

Comment:
Hmm, why do we need both arguments?

Reply:
Thanks @anvacaru, I agree, we don't need `--no-llvm-library` in `kevm kompile`. I removed it in https://github.com/runtimeverification/evm-semantics/pull/1963/commits/8e5e7f1d3abbde036752057de58e29789db541ed to follow the convention of "positive and negative flags if the default is true, just a positive flag if the default is false”, as mentioned in https://github.com/runtimeverification/evm-semantics/pull/1716#discussion_r1221932255.